Makefile

```diff
@@ -51,48 +51,72 @@ release-linux: | lambda-all llrt-linux-x64.zip llrt-linux-arm64.zip
 release-darwin: | llrt-darwin-x64.zip llrt-darwin-arm64.zip
 
 llrt-darwin-x64.zip: | clean-js js
-	cargo $(BUILD_ARG) --target $(TARGET_darwin_x86_64)
+	cargo $(BUILD_ARG) --target $(TARGET_darwin_x86_64) --features no-sdk
 	zip -j $@ target/$(TARGET_darwin_x86_64)/release/llrt
 
 llrt-darwin-arm64.zip: | clean-js js
-	cargo $(BUILD_ARG) --target $(TARGET_darwin_arm64)
+	cargo $(BUILD_ARG) --target $(TARGET_darwin_arm64) --features no-sdk
 	zip -j $@ target/$(TARGET_darwin_arm64)/release/llrt
 
 llrt-linux-x64.zip: | clean-js js
-	cargo $(BUILD_ARG) --target $(TARGET_linux_x86_64)
+	cargo $(BUILD_ARG) --target $(TARGET_linux_x86_64) --features no-sdk
 	zip -j $@ target/$(TARGET_linux_x86_64)/release/llrt
 
 llrt-linux-arm64.zip: | clean-js js
-	cargo $(BUILD_ARG) --target $(TARGET_linux_arm64)
+	cargo $(BUILD_ARG) --target $(TARGET_linux_arm64) --features no-sdk
 	zip -j $@ target/$(TARGET_linux_arm64)/release/llrt
 
 llrt-linux-x86_64.zip: llrt-linux-x64.zip
 llrt-darwin-x86_64.zip: llrt-darwin-x64.zip
 
-define release_template
-release-${1}: | clean-js js
-	cargo $$(BUILD_ARG) --target $$(TARGET_linux_$$(RELEASE_ARCH_NAME_${1})) --features lambda -vv
+define lambda_release_template
+release-${1}-full-sdk: llrt-lambda-${1}-full-sdk
+release-${1}: llrt-lambda-${1}
+release-${1}-no-sdk: llrt-lambda-${1}-no-sdk
+
+llrt-lambda-${1}-full-sdk: export SDK_BUNDLE_MODE = FULL
+llrt-lambda-${1}-full-sdk: | clean-js js
+	cargo $$(BUILD_ARG) --target $$(TARGET_linux_$$(RELEASE_ARCH_NAME_${1})) --features lambda
+	./pack target/$$(TARGET_linux_$$(RELEASE_ARCH_NAME_${1}))/release/llrt target/$$(TARGET_linux_$$(RELEASE_ARCH_NAME_${1}))/release/bootstrap
+	@rm -rf llrt-lambda-${1}-full-sdk.zip
+	zip -j llrt-lambda-${1}-full-sdk.zip target/$$(TARGET_linux_$$(RELEASE_ARCH_NAME_${1}))/release/bootstrap
+	cargo $$(BUILD_ARG) --target $$(TARGET_linux_$$(RELEASE_ARCH_NAME_${1})) --features lambda,uncompressed
+	mv target/$$(TARGET_linux_$$(RELEASE_ARCH_NAME_${1}))/release/llrt llrt-container-${1}-full-sdk
+
+llrt-lambda-${1}: export SDK_BUNDLE_MODE = STD
+llrt-lambda-${1}: | clean-js js
+	cargo $$(BUILD_ARG) --target $$(TARGET_linux_$$(RELEASE_ARCH_NAME_${1})) --features lambda
```

Comment:
I think we can reduce a lot of code by adding more variables to the template.
```
define release_template
release-${1}-${2}
```

Reply:
Please confirm that you have made the correction.